src/utils/saml.ts

```diff
@@ -140,6 +146,10 @@ const getEntrypointCerts = (
     .chain(entityID =>
       fromNullable(idps[entityID]).map(
         (idp): IEntrypointCerts => ({
+          additionalParams: {
```

Comment:
i think this is overkill. we just need one param to skip some (or all!) preValidateresponse checks
(something like: validation: "loose" <-- enum (loose, strict)). there's no need to have this kind of granularity

Reply:
there are other IDP thay may need this feature

Reply:
Yes, this is an easier solution, but we lose validation on non-strict IDP. If It isn't an issue we can use this approach.

Reply:
the goal is to selectively disable some checks. what do you mean with "we lose validatin on non-strict idp ?"

Reply:
I mean that we disable some checks on a subset of IDP without the possibility of specifying what checks are disabled.

Reply:
yes, it's not an issue : )

Reply:
fixed in 075b0a8